e2e_deployment_files/nestededge_bottomLayerBaseDeployment_connectivity_mqtt.template.json

```diff
@@ -698,95 +734,103 @@
             "TestOperationResultType": "Messages",
             "ExpectedSource": "loadGen1.send",
             "ActualSource": "relayer1.receive",
-            "TestDescription": "messages | local | amqp"
+            "TestDescription": "messages | local | amqp | nested"
```

Comment:
Rather than add nested to the test description everywhere, I would rather just add an optional metadata field to use where needed. I will explore this.

Also, tried just reading whether we were in nested from environment vars but this caused a lot of problems with the tests and would necessitate larger changes there and potentially elsewhere.